src/fragments/start/getting-started/flutter/add-api.mdx

```diff
@@ -22,9 +22,9 @@ On the creation form:
 
     ![Create app backend](/images/lib/getting-started/flutter/connect-to-cloud-create-app-backend.png)
 
-4. The on screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Launch Studio**.
+4. The on-screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Launch Studio**.
 
-    ![Open admin UI](/images/lib/getting-started/flutter/connect-to-cloud-open-admin-ui.png)
```

Comment:
Is the corresponding file still used? If not can it be deleted?

Reply:
Yes Caleb. It is not being used. I have removed it.